proposals/description/clawback.ts

```diff
@@ -0,0 +1,57 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const clawback: TemplatedProposalDescription = {
+  title: 'FIP-113: End Departed Rari Founders Vesting of TRIBE',
+  commands: [
+    /////////  Mint FEI and TRIBE onto new RARI timelocks
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: (addresses) => [addresses.newRariInfraFeiTimelock, '3254306506849315068493151'],
+      description: 'Mint FEI to the new Rari Infra FEI timelock'
+    },
+
+    {
+      target: 'core',
+      values: '0',
+      method: 'allocateTribe(address,uint256)',
+      arguments: (addresses) => [addresses.newRariInfraTribeTimelock, '3254296867072552004058854'],
```

Comment:
todo: update with the new balance after rari infra team makes a last claim

Reply:
done